### DISCUSSION TITLE
WebRTC on isolated local network

### DISCUSSION THREAD

**Discussion by LeRatierBretonnien** [bluenviron/mediamtx#3803] [Questions]
### Question

Hello here. 

I am facing a little annoying issue with WebRTC streams on isolated local network (ie no internet at all).
When requesting the stream from the browser, there is a delay of ~20-30 seconds before getting the video. Then everything is perfect, with very low latency.

So I suspect that during the first connection, something is trying to reach an external website (ICE ?) or whatever, but I am not able to figure what.

Do you have any hints or ideas to speedup initial WebRTC negotiation in the MediaMTX settings ?

Thanks !

**Comment by RedShift1** [comment]
Can you show us the configuration you're running?

**Comment by LeRatierBretonnien** [comment]
I was going to post my configuration, but during its review, I discovevered that some TURN server were declared. After deletion of this settings, and retry, it works !
So ticket can be closed !

**Reply by SMDIndomitable** [reply]
Hi, may I know where is the TURN Server declared? I seems to have this problem too, but my IceServer list is empty.

**Reply by RedShift1** [reply]
https://github.com/bluenviron/mediamtx/blob/main/mediamtx.yml#L376
